ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -71,8 +72,7 @@ metadata:
               "vipNetworkList": [
                 {
                   "networkName": "",
-                  "cidr": "",
-                  "v6cidr": ""
+                  "cidr": ""
```

Comment:
`networkUUID` field is introduced in VIP network list and nodeNetworkList.

Reply:
@akshayhavile  I have not added `networkUUID` again because we will be specifying only one of these two.